kafka/config_test.go

```diff
@@ -124,3 +124,21 @@ func TestConfigMapAPIs(t *testing.T) {
 	}
 
 }
+
+// Test that plugins will always be configured before their config options
+func TestConfigPluginPaths(t *testing.T) {
+	config := &ConfigMap{
+		"plugin.library.paths":                     "monitoring-interceptor",
```

Comment:
This will fail if the monitoring-interceptors are not installed.
I'm not sure what the best solution is here, maybe we should first try to set only plugin.library.paths to verify that the interceptor is available, and if not just skip the test.